resources/intent/weedo_x40/Extrudr/weedo_x40_0.4_Extrudr_GreenTECPro_Anthracit_Extra_Fine_Engineering.inst.cfg

```diff
@@ -0,0 +1,31 @@
+[general]
+version = 4
+name = Accurate
+definition = weedo_x40
+
+[metadata]
+setting_version = 17
+type = intent
+quality_type = extra fine
+intent_category = engineering
+material = Extrudr_GreenTECPro_Black_175
+variant = 0.4mm Nozzle
+
+[values]
+infill_sparse_density = 30
+outer_inset_first = True
+infill_line_width = =line_width
+speed_equalize_flow_enabled = True
+acceleration_enabled = True
+acceleration_print = 400
+acceleration_travel = 800
+acceleration_travel_layer_0 = =acceleration_travel / 2
+
+jerk_enabled = True
+line_width = =0.02 + 0.7 * machine_nozzle_size
+
+speed_print = 30
+top_bottom_thickness = =wall_thickness
```

Comment:
It seems that you're setting this for every intent. Shouldn't it be in the quality instead? (or should it be set like this for all intents except for default?)